scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -258,6 +258,18 @@ def run_cmake(self, args=""):
             f'-G{self.env.generator}'
         ]
 
+        if self.testsuite.sysbuild:
+            logger.debug("Building %s using sysbuild" % (self.source_dir))
+            source_args = [
+                f'-Sshare/sysbuild',
```

Comment:
```suggestion
                f'-S{canonical_zephyr_base}/share/sysbuild',
```
to make it work from outside of Zephyr root directory.

Reply:
fixed in latest push